### ISSUE TITLE
[A11Y] - OMP Reader View | Heading on main page is not visible and lacks better description for purpose of the page

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12659]
## Issue Description
As part of our on-going commitments to accessibility we are curating the issues that the auditors have highlighted while analyzing our systems.  This would affect all users.

## How to reproduce / Current behavior
Go to OMP demo site main page `/index.php/aup-demo` and search for the `<h1>` tag.  it is not visible.  Additionally the auditor notes in general that headings should “Clearly say what the website is and who it’s for as soon as you land on the page.”   

## What application are you using?
 OMP 3.5.0-4

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: latest

## Testing method
Manual, using screenreader

## Action
Apply a proper `<h1> tag` with a good description of the page.  Not exactly sure what a good description would be but it should describe the page's purpose.  Offered a suggestion for the description of h1 tag.
- [ ] "Book of the month, displaying featured and new release books for the month"

## Additional information

<img width="1089" height="48" alt="Image" src="https://github.com/user-attachments/assets/8e4e20e5-50f8-48d4-9405-73878a6cdf6a" />
